loculus_values/environment_specific_values/production.yaml

```
@@ -4,6 +4,7 @@ runDevelopmentKeycloakDatabase: false
 bannerMessage: ""
 robotsNoindexHeader: false
 subdomainSeparator: "."
+bannerMessage: "Some portions of our website are experiencing intermittent outages, and some requests may take longer than normal to complete. We apologize for the inconvenience and thank you for your patience as we resolve the issue."
```

This sounds like unplanned issues
```suggestion
bannerMessage: "We are rolling out a new version of the Pathoplexus processing pipeline. Sequences may take longer to process and there may be downtime of our query API. We apologise for any inconvenience."
```